hdl/projects/cosmo_seq/cosmo_seq_top.rdl

```diff
@@ -14,10 +14,10 @@ addrmap cosmo_seq_top {
     // Instantiate 2 to test nesting
     info_regs info @ 0x0;
     spi_nor_regs spi_nor @ 0x0100;
-    espi_regs espi @ 0x0200;
-    sequencer_regs sequencer @ 0x300;
-    sp_i2c_regs sp_i2c @ 0x0400;
-    pca9506_axi_regs fpga1_hotplug @ 0x0500;
-    dimm_regs dimms @ 0x0600;
-    debug_regs debug_ctrl @ 0x0700;
```

Comment:
Maybe just a future piece of food for thought, but should we make the default space we are allocating bigger than 8-bits wide? Like perhaps defaulting to peripherals with 12 or 16-bit address spaces?We can't see the future, and 256-bytes has been fine for all sorts of things, but as soon as you want a RAM it all blows up. This is not an immediately actionable thought.